docs/usage/dataset.rst

```diff
@@ -65,10 +69,33 @@ in the dataset:
 
 .. code:: python
 
-	dataset = EpitomeDataset()
+	dataset = EpitomeDataset(data_dir="configured_data_path")
 
 	dataset.list_targets() # prints > 200 ChIP-seq targets
 
 
 
 You can now use your ``dataset`` in an Epitome model.
+
+Load your processed dataset
+-------------------------
+You can specify the data path and/or genome assembly that you would like to use
+in the Epitome dataset. You just need to define the ``data_dir`` and/or
+``assembly`` variables.
+
+.. code:: python
+
+	dataset = EpitomeDataset(data_dir="~/$USERNAME/epitome/data",
+				assembly="hg19")
+
+Note if both the ``data_dir`` and ``assembly`` are set, the dataset will
+append the specified assembly to the data_dir and fetch the data located in there
+(~/$USERNAME/epitome/data/hg19/data.zip).
+
+You do not need to define both variables though. If you leave ``data_dir`` empty,
+the Epitome dataset will look for a valid default data in ``~/$USER_NAME/.epitome/data/``
```

Comment:
can you provide code examples for each case, then explain what happens in a sentence or two, instead of including a block of text that includes all cases

Reply:
Done